sklearn/linear_model/_ridge.py

```diff
@@ -1905,3 +1905,11 @@ def fit(self, X, y, sample_weight=None):
     @property
     def classes_(self):
         return self._label_binarizer.classes_
+
+    def _more_tags(self):
+        return {
+            '_xfail_test': {
+                'check_sample_weights_invariance(kind=zeros)':
+                'zero sample_weight is not equivalent to removing samples',
+            }
+        }
```

Comment:
RidgeClassifierCV now also started to fail with a quite high tolerance.

Reply:
> Completely unrelated to this PR but since you've been doing this (which is helpful, thanks): do you think it'd be useful for github to support comments from PR author to reviewers, but something distinct from regular review comment?

Maybe, but I can't say the current way it works is an issue for me.